test/puppetlabs/ssl_utils/core_test.clj

```diff
@@ -253,6 +253,20 @@
                                         "2.5.29.35")]
         (is (= actual-ext expected-ext))))
 
+    (testing (str "signing as an intermediate ca with a type 2 subject id and with a certificate")
+      (let [int-sign-exts (-> (create-ca-extensions (get-public-key issuer-key-pair) (get-public-key key-pair))
+                              (swap-auth-key-identifier (get-public-key key-pair)))
```

Comment:
I think we want to swap the type of subject key id on the intermediate right?

Reply:
oh, yeah, that's right, good catch @justinstoller !